### ISSUE TITLE
Migration - Extra escaping after running migration

### ISSUE THREAD

**Issue by YujiShen** [phalcon/phalcon-devtools#1137]
### Expected and Actual Behavior

I except the migration data will generate same result as it should be.

> **Describe what you are trying to achieve and what goes wrong.**

A VARCHAR filed before migration:
```
"I was so ... advisors."
```

A TEXT filed before migration:
```
{"title":"\u5b66\u672f\u8f85\u5bfc","learnmore_title":"\u67e5\u770b\u66f4\u591a","learnmore_href":{"type":"links","content":"https:\/\/www.xxx.com\/services\/"},"image":"\/public\/files\/2017\/10\/20171019_51fc0791176f58906c690aba9c1d5dc3.png","image_alt":"\u539a\u4ec1","refs":{"1":{"table":"file","id":"17","column":"header"}}}
```
> Provide output if related

VARCHAR filed after migration:
```
\"I was so ... advisors.\"
```

TEXT filed after migration:
```
{\"title\":\"\\u5b66\\u672f\\u8f85\\u5bfc\",\"learnmore_title\":\"\\u67e5\\u770b\\u66f4\\u591a\",\"learnmore_href\":{\"type\":\"links\",\"content\":\"https:\\/\\/www.xxx.com\\/services\\/\"},\"image\":\"\\/public\\/files\\/2017\\/10\\/20171019_51fc0791176f58906c690aba9c1d5dc3.png\",\"image_alt\":\"\\u539a\\u4ec1\",\"refs\":{\"1\":{\"table\":\"file\",\"id\":\"17\",\"column\":\"header\"}}}
```
> Provide minimal script to reproduce the issue

Here is the command I use to migrate
```sh
# generate
phalcon migration generate --config=backend/core/starter/config/config.yaml --migrations=backend/migrations --data=always

# run
phalcon migration run --config=backend/core/starter/config/config.yaml --migrations=backend/migrations
```

config.yaml
```yaml
database:
  adapter: mysql
  host: xxx
  username: xxx
  password: xxx
  dbname: xxx
```

Actually, I also encounter same escaping problem when I use raw sql in Phalcon to update that TEXT filed (PHP array after json_encode()), but there is no problem if I use model->save(). The data before migration are inserted by model->save().

### Details
* System info and versions (if possible): (`phalcon info`)
```
Environment:
  OS: Linux 382c971c7d8e 4.9.49-moby #1 SMP Wed Sep 27 23:17:17 UTC 2017 x86_64
  PHP Version: 7.1.9
  PHP SAPI: cli
  PHP Bin: /usr/bin/php7
  PHP Extension Dir: /usr/lib/php7/modules
  PHP Bin Dir: /usr/bin
  Loaded PHP config: /etc/php7/php.ini
Versions:
  Phalcon DevTools Version: 3.2.8
  Phalcon Version: 3.2.4
  AdminLTE Version: 2.3.6
```
* Phalcon Framework version: (`php --ri phalcon`)
```
Web framework delivered as a C-extension for PHP
phalcon => enabled
Author => Phalcon Team and contributors
Version => 3.2.4
Build Date => Oct 26 2017 13:04:27
Powered by Zephir => Version 0.10.4-11e39849b0

Directive => Local Value => Master Value
phalcon.db.escape_identifiers => On => On
phalcon.db.force_casting => Off => Off
phalcon.orm.events => On => On
phalcon.orm.virtual_foreign_keys => On => On
phalcon.orm.column_renaming => On => On
phalcon.orm.not_null_validations => On => On
phalcon.orm.exception_on_failed_save => Off => Off
phalcon.orm.enable_literals => On => On
phalcon.orm.late_state_binding => Off => Off
phalcon.orm.enable_implicit_joins => On => On
phalcon.orm.cast_on_hydrate => Off => Off
phalcon.orm.ignore_unknown_columns => Off => Off
phalcon.orm.update_snapshot_on_save => On => On
phalcon.orm.disable_assign_setters => Off => Off
```

* Operating System: Alpine Linux
* Server: Nginx
* Other related info (Database, table schema): MariaDB 10.1.26

**Comment by sergeysviridenko** [comment]
@YujiShen I'll try to sort out with this as soon as possible. Thank you for report about it.

**Comment by sergeyklay** [comment]
Fixed in the `3.2.x` branch. Feel free to open a new issue if the problem appears again. Thank you for contributing.
